MatrixSDK/VoIP/MXCallManager.m

```diff
@@ -1267,6 +1387,199 @@ - (void)placeCallAgainst:(NSString *)phoneNumber
     } failure:failure];
 }
 
+#pragma mark - Virtual Rooms
+
+- (void)setVirtualRoomsSupported:(BOOL)virtualRoomsSupported
+{
+    _virtualRoomsSupported = virtualRoomsSupported;
+    
+    [[NSNotificationCenter defaultCenter] postNotificationName:kMXCallManagerVirtualRoomsSupportUpdated
+                                                        object:self];
+}
+
+- (void)getVirtualUserFrom:(NSString *)userId
+                   success:(void (^)(MXThirdPartyUserInstance * _Nonnull))success
+                   failure:(void (^)(NSError * _Nullable))failure
+{
+    [_mxSession.matrixRestClient thirdpartyUsers:kMXProtocolVectorSipVirtual
+                                          fields:@{
+                                              @"native_mxid": userId
+                                          }
+                                         success:^(MXThirdPartyUsersResponse *thirdpartyUsersResponse) {
+        
+        MXThirdPartyUserInstance * user = [thirdpartyUsersResponse.users firstObject];
+        
+        NSLog(@"Succeeded to look up the virtual user: %@", user.userId);
```

Comment:
Can you a prefix tag like `[MXCallManager] getVirtualUserFrom: `, please?

Reply:
👍